### PR TITLE
rebuild: add functest, fix help message

### PR DESCRIPTION
**Pull request by swiftgist**
Signed-off-by: Eric Jackson <ejackson@suse.com>

-----------------

**Checklist:**
- [x] Added unittests and or functional tests
- [ ] Adapted documentation
- [ ] Referenced issues or internal bugtracker
- [ ] Ran integration tests successfully (trigger with "@susebot run teuthology" in a GitHub comment; see the [wiki](https://github.com/SUSE/DeepSea/wiki/Testing) for more information)

### CODE REVIEW

**Comment by swiftgist** [comment]
@susebot run teuthology

**Comment by jschmid1** [comment]
1) linter

```
21:26:45 ************* Module srv.modules.runners.rebuild
21:26:45 W:254,17: Duplicate key 'node' in dictionary (duplicate-key)
```

2) functests

It looks like the current 'functest' testrun fails due to the missing alertmanager rpm (from #1650) 
Only the functest test fails with this error, which is rather weird..

3) Code

By the looks of it, the rebuild functest is not enabled. See https://github.com/SUSE/DeepSea/pull/1658/files#diff-9d0ae1f9554b0c101b9817e7366e68b2R15

**Comment by smithfarm** [comment]
jschmid1: Is the linter failure something we can ignore?

File: srv/modules/runners/rebuild.py [hunk 1]
```diff
@@ -254,4 +254,5 @@ def check(*args):
 __func_alias__ = {
                  'help_': 'help',
                  'node': 'nodes',
+                 'node': 'node',
```
Comment (smithfarm) [inline srv/modules/runners/rebuild.py]: linter caught this -> the key `'node'` is duplicated

**Comment by smithfarm** [comment]
> It looks like the current 'functest' testrun fails due to the missing alertmanager rpm (from #1650)

This has since been fixed.

> Only the functest test fails with this error, which is rather weird..

`deepsea/tier2/functional` is the only test case in tier2 which has prometheus and grafana roles enabled in policy.cfg

**Comment by swiftgist** [comment]
I added the .rebuild to the init.sls, but it's commented out.  When I saw the other entries commented, I thought there may be a current issue.  So, I don't see how this could even get called at the moment.

**Comment by jschmid1** [comment]
rebuild is still disabled.. leaving that for later

**Pull request merged**